client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js

```diff
@@ -179,6 +194,31 @@ const convertToApiPackage = pckg => {
 		'weight',
 		'signature',
 	] );
+	if ( customsItems ) {
+		apiPckg.contents_type =
+			'other' !== pckg.contentsType ? pckg.contentsType || 'merchandise' : pckg.contentsExplanation;
+		apiPckg.restriction_type =
+			'other' !== pckg.restrictionType
+				? pckg.restrictionType || 'none'
+				: pckg.restrictionExplanation;
+		apiPckg.abandon_on_non_delivery = Boolean( pckg.abandonOnNonDelivery );
+		apiPckg.itn = pckg.itn || '';
+		const getProductValue = productId =>
+			isNil( customsItems[ productId ].value )
+				? getProductValueFromOrder( state, productId, orderId, siteId )
+				: customsItems[ productId ].value;
```

Comment:
Why do we need `getProductValueFromOrder()` and `customsItems[ productId ].value` here?

To add more context to my question:
- `getProductValueFromOrder()` get the product value from the order line items, `order.line_items`. We get those using the selector `getOrder` - a WooCommerce selector (not services-specific)
- `customsItems[ productId ].value` is only set once, when initializing the form data for printing the label in https://github.com/Automattic/wp-calypso/pull/24974/files#diff-45c5247f05768aa908f6e59f26bd6d3dR61
To initialize the form, we fetch the labels data in https://github.com/Automattic/wp-calypso/blob/a57cb53c56b891bf55d29f74cd8595a3f77687fd/client/extensions/woocommerce/woocommerce-services/state/shipping-label/actions.js#L105

Can one of these values serve as the single source of truth? If not, can we make one of them serve as the single source of truth in our case?